src/pug/includes/body.pug

```diff
@@ -17,18 +17,19 @@ include ./fork.pug
                     .select-wrapper
                         label.sr-only(for="filter") Filter opportunity difficulty
                         select#filter
-                            for filter in ['All difficulties', 'Easy', 'Medium', 'Hard']
-                                option(value=filter.replace(/ /g, '').toLowerCase()) #{filter}
+                            for filter in ['All', 'Easy', 'Medium', 'Hard']
```

Comment:
I think we should lowercase this as well as the sort-order input to make it more like a choose-your-own journey